### ISSUE TITLE
Avoid inv()

### ISSUE THREAD

**Issue by fbastin** [chkwon/jpor_codes#1]
You should not compute x_B as x_B = inv(B)*b, but as the solution of a linear system: x_B = B\b, and check that the rank of B is equal to m

**Comment by chkwon** [comment]
Indeed. `inv(B)` was used just for education purpose. Perhaps I should also add `B\b`. Thanks.
